dependencies.yaml

```diff
@@ -46,7 +46,7 @@ libfsm:
 # libiso15118
 libiso15118:
   git: https://github.com/EVerest/libiso15118.git
-  git_tag: v0.2.0
+  git_tag: a135d7d3cc62a2206b1128b267a388e209b3cd2c
```

Comment:
This should point to a release before merge

Reply:
This will change when the libiso15118 dynamic part will be merged and a new libiso version is published